models/auth/oauth2.go

```diff
@@ -59,10 +60,12 @@ func (app *OAuth2Application) ContainsRedirectURI(redirectURI string) bool {
 
 // GenerateClientSecret will generate the client secret and returns the plaintext and saves the hash at the database
 func (app *OAuth2Application) GenerateClientSecret() (string, error) {
-	clientSecret, err := secret.New()
+	rBytes, err := util.CryptoRandomBytes(44)
```

Comment:
Do we really need that long?  AES key size is only 16 or 32 bytes .....

Reply:
> Do we really need that long? AES key size is only 16 or 32 bytes .....

I've copied the length from `secret.New()`, we could reduce it to 32, that still provides enough security than the current code.

Reply:
Hmm, we might even want to add a prefix? Such that security scanners can easily grep such tokens in source codes. Github has done it as well: https://github.blog/2021-04-05-behind-githubs-new-authentication-token-formats/

Reply:
Then we can leave a comment in code to describe why there is a prefix 😊